README.md

```diff
@@ -64,7 +64,7 @@ See the [documentation](http://xeus.readthedocs.io/) for an exhaustive list of t
 
 | xeus-zmq |  xeus   | ZeroMQ  | cppzmq  | nlohmann json | OpenSSL |      xtl       |
 |----------|---------|---------|---------|---------------|---------|----------------|
-| master   | ^3.1.1  | ^4.2.5  | ^4.8.1  |     ^3.2.0    | ^3.0    | >=0.7.0,<0.8.0 |
+| master   | ^4.0.0  | ^4.2.5  | ^4.8.1  |     ^3.11.3   | ^3.0    | >=0.7.0,<0.8.0 |
```

Comment:
Should we finally rename this to `main`?

Reply:
Oh indeed, good catch!